### ISSUE TITLE
Sheep not spawning

### ISSUE THREAD

**Issue by malnaur** [anegostudios/VintageStory-Issues#3786]
### Game Version

v1.19.7

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Started a new game when 1.19.6 came out and I have done a lot of wandering. I've trapped and domesticated chickens and pigs but I've come to realize I pretty much NEVER see sheep. They used to be everywhere. This has to be a bug. I see other deer, moose, and goats but no sheep. In fact, animals are much more scarce than they were except for wolves and bears, of course.

I have set the world to 25kx25k with 10k equator to pole distance but prior worlds with the same settings had plenty of animals.

The wiki says the spawn is minimum 140% of sealevel at start and 130% of sealevel at runtime. Nowhere is a spec for sealevel (thanks) other than the windmill entry (!) which states normal is 110. If so, 130% of that is 143. Such terrain does not exist other than very isolated locations where that altitude spawns glacial ice! Normal mountains are maybe 125-130 high. If this is new, it is BAD. Sheep were plentiful, now they are extinct. Make sheep great again! As the only source of dairy and given the aggravation of making cheese, few players will scour the map hoping to find enough lambs to get a mating pair, trapping them, and then hauling them home.

### How to reproduce

Look for sheep in a 1.19.7 world., I dare you.

_No response_

### Logs

```Log```

**Comment by malnaur** [comment]
OK. I found 'some' on ridiculous mountaintops that were mostly barren or held a single animal. Once I found a breeding pair, I could painfully build a herd THERE and eventually trap enough lambs to be worth bringing back (of course, max 3 or 4 at a time). This is not useful for solo play without teleport to cheese the search and the transport. At that point, I might as well just spawn a pair at home. This may be more 'realistic' but it does not improve gameplay except for diehards.
